README.md

```diff
@@ -1,3 +1,19 @@
+<a href="https://exograph.dev">
+  <p align="center">
+    <picture>
+      <source media="(prefers-color-scheme: dark)" srcset="logo-dark.svg">
```

Comment:
You'll (sadly) need to use an absolute path here. Also I wonder if SVGs are even supported.

Reply:
It seems to work fine: https://github.com/exograph/exograph/tree/update-readme (also checked on Safari)

Reply:
It's failing to load for me on mobile. The issue might be that SVGs are not supported in that browser or something.

Reply:
What's the browser out of curiosity? I think svg should be Ok on most things https://caniuse.com/?search=svg.

Reply:
It was the one inside the Github app. It turns out the PNG version doesn't work, either.